### ISSUE TITLE
ui5-wizard: [AXE] ARIA attributes must conform to valid values

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#3160]
**Describe the bug**
Ensures all ARIA attributes have valid values.
Invalid ARIA attribute value: aria-controls="ui5-wiz-content"

**Screenshots**
<img width="1791" alt="2021-04-19_13-20-34" src="https://user-images.githubusercontent.com/13098801/115185306-24928f00-a112-11eb-8732-a861e62f7f50.png">

**Context**
- UI5 Web Components version: 0.29.14
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

Organization:
SF

Priority: (optional) (Low, Medium, High or Very High)
High

Please fix this issue in 0.29.15

**Comment by alexandar-mitsev** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Can you take a look at this issue. I think the aria-controls property used in the wizard should point to an id.

Regards,
Alex

**Comment by elenastoyanovaa** [comment]
Dear colleagues,

The aria-controls is set to an element with a class equal to "ui5-wiz-content", but aria-controls receives ID, not classes. This should be adjusted.

Kind Regards,
Elena

**Comment by ilhan007** [comment]
Hello @GongRichard  The fix is released with 0.29.15
